app/src/main/java/org/wikipedia/WikipediaApp.kt

```diff
@@ -79,13 +78,7 @@ class WikipediaApp : Application() {
         }
 
     val appOrSystemLanguageCode: String
-        get() {
-            val code = languageState.appLanguageCode
-            if (AccountUtil.getUserIdForLanguage(code) == 0) {
-                getUserIdForLanguage(code)
```

Comment:
If the getter for `appOrSystemLanguageCode` is called in quick succession from multiple places, it will kick off multiple unnecessary requests.
And my next question is, where exactly do we use the UserID?